backend/databasemodel/model.sql

```diff
@@ -1406,39 +1417,50 @@ CREATE TABLE kooste.osm_alueet (
 	osm_id bigint NOT NULL,
 	osm_type text NOT NULL,
 	geom geometry(POLYGON, 4326) NOT NULL,
+	tarmo_category text NOT NULL,
 	tags jsonb,
 	CONSTRAINT osm_alueet_pk PRIMARY KEY (id),
 	UNIQUE (osm_id, osm_type)
 );
-ALTER TABLE kooste.osm_pisteet OWNER TO tarmo_admin;
+ALTER TABLE kooste.osm_alueet OWNER TO tarmo_admin;
 
 -- object: kooste.osm_metadata | type: TABLE --
 -- DROP TABLE IF EXISTS lipas.metadata CASCADE;
 CREATE TABLE kooste.osm_metadata (
-	update_id bigint NOT NULL GENERATED ALWAYS AS IDENTITY ,
+	update_id bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
 	last_modified timestamptz,
 	tags_to_include jsonb,
 	tags_to_exclude jsonb,
+	osm_types_tarmo_joukkoliikennepysakit jsonb,
```

Comment:
What is this? We don't use OSM for joukkoliikennepysäkit, do we intend to?

Reply:
My mistake; see my other comments. I will remove this osm_types_tarmo_joukkoliikennepysäkit field and restore this table to its previous state.